### DISCUSSION TITLE
Add latest tag on docker push

### DISCUSSION THREAD

**Discussion by rmoreas** [helmfile/helmfile#753] [Ideas]
The `latest` tag is missing on the container repository `ghcr.io/helmfile/helmfile`.
A `latest` tag should be pushed on each new release.

**Comment by yxxhero** [comment]
make sense. PR is welcome.

**Reply by yxxhero** [reply]
@rmoreas

**Reply by rmoreas** [reply]
Would do, but I'm not so familiar with Github actions, I'm a Gitlab user 🙂

**Reply by yxxhero** [reply]
very similar. please free to do that.

**Comment by rmoreas** [comment]
Is see that there is a custom `setup-image-vars` action. Why not use the [`metadata-action`](https://github.com/docker/metadata-action), this one already takes care of automatically adding the latest tag.

**Comment by yxxhero** [comment]
please free to update.

**Reply by rmoreas** [reply]
added PR #756, but not sure how to test it.

**Reply by yxxhero** [reply]
tested in ci.
